src/stylesheets/elements/_buttons.scss

```diff
@@ -122,7 +94,7 @@ button,
     }
   }
 
-  &.usa-button-outline-inverse {
+  &.usa-button-secondary-inverse {
```

Comment:
This looks like a breaking change right? Because if somebody has a button with outline-inverse, then the button won't be targeted for those rules anymore? Did you have thoughts on whether it's OK to break this?

If we feel that it's necessary, we can keep the code the way you have it, and add a "Deprecated" section at the bottom of the file that has the rules to ensure nothing will break. Then we can remove the deprecated section once we upgrade to 2.0?